components/Breadcrumb/src/index.tsx

```diff
@@ -108,19 +72,26 @@ export const Breadcrumb: React.FC<BreadcrumbProps> = (props: BreadcrumbProps) =>
         ) : (
           <BreadcrumbItem item={item} isLastItem={isLastItem} contentNumber={contentNumber} />
         )}
-      </li>
+        <meta content={contentNumber.toString()} itemProp="position" />
```

Comment:
Hmm I'm wondering if the `contentNumber` as it's content would already be enough? Or is it the typing requiring us to explicitly convert it to a string?

Reply:
Yes, typing requires us to explicitly convert the contentNumber to a string.